server.js

```diff
@@ -36,6 +37,11 @@ function sendWeatherData(request, response){
   let weatherArray =[];
   const jsonWeatherObj = require('./data/weather.json');
   
+//==error message==
+  // if(request.query.forecast !== ''){
+  //   return response.status(500).send('');
+  // }
+//==================
```

Comment:
Sending out a error message is still a confusing topic for me, especially since I kept reaching an error within my error message function.